extensions/extensions-helm/aissemble-data-access-chart/values.yaml

```diff
@@ -9,13 +9,11 @@ aissemble-quarkus-chart:
         protocol: TCP
         targetPort: 8080
   deployment:
-    ports:
-      - name: http-1
-        containerPort: 8081
```

Comment:
Q: Unused?

Reply:
Correct. The quarkus chart sets the deployment ports using the `service.ports`, so this attribute it never actually used. Good catch though, the env below should have the name updated to `http`.